doc/developer/documenting.rst

```diff
@@ -34,4 +34,34 @@ documentation is built. To do this, you can clean the output with::
 	jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace mynotebook.ipynb
 
 This should clear the output and overwrite the file. If this doesn't work, you can clear
-all output cells in the notebook web interface itself before committing the file.
\ No newline at end of file
+all output cells in the notebook web interface itself before committing the file.
+
+Documentation for ARMI plugins
+------------------------------
+
+Linking to ARMI documentation from plugins
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ARMI plugin documentation can feature rich hyperlinks to the ARMI API documentation 
+with the help of the 
+`intersphinx plugin <http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_.
+The plugin documentation config file simply should add ``"sphinx.ext.intersphinx",`` to
```

Comment:
Consider removing "simply" (A little opinionated, but I do agree with the sentiment: https://www.knowledgeowl.com/home/dont-say-simply-jim-fisher)

Reply:
Done. thanks. haha

Reply:
awww man you can't even say "boom, you're done"!?